File: src/test/kotlin/study/StringTest.java
package study;

import org.junit.jupiter.api.Test;

import static org.assertj.core.api.Assertions.assertThat;

public class StringTest {

    @Test
    void isBlank() {
        assertThat(" ".isBlank()).isTrue();
        assertThat("\n".isBlank()).isTrue();
        assertThat("\t".isBlank()).isTrue();
        assertThat("""\n""".isBlank())
    }

}

File: src/test/kotlin/study/StringTest.kt
package study

import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

class StringTest {
    @Test
    fun isBlank() {
        assertThat(" ".isBlank()).isTrue()
        assertThat("\n".isBlank()).isTrue()
        assertThat("\t".isBlank()).isTrue()
        assertThat("""\n""".isBlank()).isFalse()
    }
}
